deploy/awstools/awstools.py

```diff
@@ -43,6 +44,11 @@ def aws_resource_names():
         'runfarmprefix':     None,
     }
 
+    # The code below assumes we're running on an AWS instance. When running on a CI instance,
+    # early out with the default dictionary
+    if os.environ.has_key("CIRCLE_WORKFLOW_ID"):
```

Comment:
Why is this needed? Won't you need this in the future when running normal FireSim FPGA simulations? Or maybe I'm misunderstanding what this is?